testsuite/ui/multinode/add_node_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package multinode
+
+import (
+	"testing"
+
+	"github.com/go-rod/rod"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/private/testplanet"
+	"storj.io/storj/testsuite/ui/uitest"
+)
+
+func TestAddNewNodeButton(t *testing.T) {
+	uitest.Multinode(t, 1, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, browser *rod.Browser) {
+		startPage := planet.Multinodes[0].ConsoleURL() + "/add-first-node"
+		page := openPage(browser, startPage)
+
+		node := planet.StorageNodes[0]
+		node2 := planet.StorageNodes[1]
+
+		page.MustElement("input#Node\\ ID.headered-input").MustInput(node.ID().String())
+		page.MustElement("input#Public\\ IP\\ Address.headered-input").MustInput(node.Addr())
+		page.MustElement("input#API\\ Key.headered-input").MustInput(node.APIKey())
+
+		addNodeButton := page.MustElement(".add-first-node__left-area__button")
+		require.Equal(t, "Add Node", addNodeButton.MustText())
+		addNodeButton.MustClick()
+		page.MustWaitNavigation()
+
+		nodesPageTitle := page.MustElement("h1.my-nodes__title").MustText()
+		require.Equal(t, "My Nodes", nodesPageTitle)
+
+		// Test- Clicks on new node button and checks if it opens up add new node modal
+		newNodeButton := page.MustElementX("//span[contains(text(),'New Node')]")
+		require.Equal(t, "New Node", newNodeButton.MustText())
+		newNodeButton.MustClick()
+		addNewNodeTitle := page.MustElement("h2:nth-child(1)").MustText()
+		require.Equal(t, "Add New Node", addNewNodeTitle)
+
+		// Test- Cancels the addition of new node in dashboard by clicking X button to cancel
+		xCancelButton := page.MustElement("div.modal__cross > svg:nth-child(1)")
+		xCancelButton.MustClick()
+		require.Equal(t, "My Nodes", nodesPageTitle)
```

Comment:
this check looks wrong

Reply:
Changed the check so now it checks for the new node modal that should disappear once its closed